content/en/docs/tasks/security/authorization/authz-ingress/test.sh

```diff
@@ -22,7 +22,7 @@ set -o pipefail
 # @setup profile=default
 
 # Set retries to a higher value because config update is slow.
-export VERIFY_RETRIES=10
+export VERIFY_RETRIES=15
```

Comment:
This will take way too long to time out. With the exponential backoff, 15 retries will take about 8 hours before failing. 10 retries is about the longest I would ever go (about 17 mins).

Reply:
Well, the test bucket will fail at 60 mins, but maybe we can increase the retries by 1 or 2. Not sure if there is something else we can do that may get the config written so that the tests do pass.

Reply:
You could bump it to 11 (about 35 mins), but I doubt that waiting more than 17 mins will help. It should never need half an hour to propagate config. My guess is there must be some kind of timing bug where the config change gets missed in which case waiting won't help.

Reply:
Making this DNM for now. Maybe can figure out if there is something we can do to get the config written and may need doc updates, etc.